src/frontend/src/components/CreateGallery.tsx

```diff
@@ -0,0 +1,133 @@
+            <div className="flex flex-col">
+              <span className="mr-2">Description</span>
+              {/* <input type="text" className="p-1 border border-gray-500 rounded max-w-xl" /> */}
+              <textarea rows={5} className="mt-2 p-1 border border-gray-500 rounded max-w-xl" />
+            </div>
+          </div>
+
+          <div className="mb-4 flex items-center">
+            <div className="flex flex-col">
+              <span className="mr-2">Hours</span>
+              <div className="items-center">
+                <input type="text" className="mt-2 p-1 border border-gray-500 rounded w-20" />
+                <span className='ml-3 mr-5'>Hrs</span>
+              </div>
+            </div>
+
+            <div className="flex flex-col">
+              <label className="mr-2">Location</label>
+              <div className="relative w-4/5">
+                <input type="text" className="mt-2 p-1 border border-gray-500 rounded w-full" />
+                {/* <div className="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none"> */}
+                <div className="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none">
+                  <GrLocation />
+                </div>
+              </div>
+            </div>
+
+            <div className="flex flex-col">
+                <span className="mr-2">Delivery time</span>
+              <div>
+                <input type="text" className="mt-2 p-1 border border-gray-500 rounded w-20" />
+                <span className="ml-3 mr-5">Days</span>
+              </div>
+            </div>
+          </div>
+
+          <div className="mb-4">
+            <div className="flex flex-col">
+              <span className="mr-2">What includes in this package</span>
+              <input type="text" className="mt-2 p-1 border border-gray-500 rounded" />
+            </div>
+
+            <div className="flex items-center">
+              <button className="text-yellow-600 flex items-center">
+                <IoIosAdd style={{ fontSize: '2rem' }}/>
+                Click to add
+              </button>
+            </div>
+
+          </div>
+
+          <div className="mb-4 flex items-center">
+            <div className="flex flex-col ">
+              <span className="mr-2">Price</span>
+              <div className="items-center">
+                <input type="text" className="mt-2 p-1 border border-gray-500 rounded w-20" />
+                <span className="mx-3">THB</span>
+              </div>
+            </div>
+          </div>
+
+        </div>
+        <div className='down-right-section mr-12'>
+          <div className="mb-4 flex justify-end">
+            <div>
+              <button className="text-white rounded px-4 py-1 mr-4" style={{ backgroundColor: '#D4D4D4' , minWidth: '5rem' }}>Cancel</button>
+              <button className="text-white rounded px-4 py-1" style={{ backgroundColor: '#E19007' , minWidth: '5rem' }}>Save</button>
```

Comment:
Use exisitng "btn btn-primary" tailwind class for consistency (more information in global.css)